azure-pipelines.yml

```diff
@@ -197,6 +197,10 @@ stages:
       parameters:
         testName: 'windows-netframework'
         workingDirectory: 'integrationtest/TargetProjects/NetFramework/FullFrameworkApp.Test'
+    - script: $(Agent.BuildDirectory)/tools/dotnet-stryker
```

Comment:
I see the empty test project is dotnet6. The windows integration test is the slowest, it would be better to add this to the linux integration test phase as it's the quickest.

Reply:
seems ok